frontend/src/components/elements/Audio/Audio.tsx

```diff
@@ -17,6 +17,8 @@
 
 import React from "react"
 import { Map as ImmutableMap } from "immutable"
+import { getWindowBaseUriParts as get_base_uri_parts } from "../../../lib/UriUtil"
+import { buildHttpUri } from "../../../lib/UriUtil"
```

Comment:
You could simplify these imports in this way 
```javascript
import { getWindowBaseUriParts as get_base_uri_parts, buildHttpUri } from "lib/UriUtil"
```

Reply:
OK!  thanks!